demo/sections/workers/WorkersTable.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <ComponentPage title="Worker Table">
+    <WorkersTable :worker-pool-name="workerPool.name" />
+  </ComponentPage>
+</template>
+
+<script lang="ts" setup>
+  import { WorkersTable } from '@/components'
+  import ComponentPage from '@/demo/components/ComponentPage.vue'
+  import { useWorkerPoolMock } from '@/demo/compositions/useWorkerPoolMock'
+  import { useWorkersMock } from '@/demo/compositions/useWorkersMock'
+
+  const workerPool = useWorkerPoolMock()
+
+  useWorkersMock(5, () => ({
+    workerPoolId: workerPool.id,
+  }))
```

Comment:
I think it be better here to use a `useWorkerPoolMock` which automatically creates some related workers. That way no overrides are needed.

Reply:
`useWorkerPoolMock` doesn't create workers. I can look into it deeper in separate PR for docs improvement, but for rn purposes I thinks this is fine

Reply:
Yep this works. And being able to do stuff like this makes me feel like the api mocks and compositions are working well. 

Just generally we want to mock the thing the component needs (the worker pool) and have the composition generate any relational data that thing could have. 

No change needed